### DISCUSSION TITLE
Any way to customize options?

### DISCUSSION THREAD

**Discussion by Tansoric** [awSweeney/FFXIV-ACT-Rainbow-Mage-Overlays#21] [General]
With previous versions of RainbowMage, you could open the files in notepad and turn certain columns off and on, as well as adjust column spacing, text size, and opacity. When I open these new files it looks like its just referencing some javascript and I can't edit anything. Is there any way to customize this?

**Comment by awSweeney** [comment]
Part of the 1.1 update was to break things apart into their respective files, rather than being all contained within one. Specifically styles/css were broken up into css files within the css directory, and javascript into the js files in the src directory. You should be able to open both these file types in notepad, or your text editor of choice. Though you may need to do an 'open with' for your computer to recognize so.

To locate which files you want to edit, opening the html file for your overlay of choice you should see the following near the top:

`	<head>`
`		<meta charset="utf-8" />`
`		<title></title>`
`		<link rel="stylesheet" href="../css/MiniParseGlow_CritMonitor_Classic.css">`
`		<script src="../lib/jquery-2.1.1.min.js" type="text/javascript"></script>`
`		<script src="../src/config.js" type="text/javascript"></script>`
`		<script src="../src/MiniParseGlow_CritMonitor_Classic.js" type="text/javascript"></script>`
`	</head>`

In this example the following two are the items you would want to edit:

{Rainbow mage directory}/css/MiniParseGlow_CritMonitor_Classic.css
{Rainbow mage directory}/src/MiniParseGlow_CritMonitor_Classic.js

If you have edited the overlay before you should now see some familiar territory. The more stylistic items you'll find in the css file. The table structure, columns etc would be found within the JS file.

If you want to avoid updates overwriting your changes in the future you can save the css/js files as something new. Copy the base overlay html and also rename it. From there change the file paths in the < head > section to your new files.

Hopefully this is what you're looking for.
